Cookbook/Current.md

```diff
@@ -0,0 +1,22 @@
+# How to use `Current` in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global implementation of the `World` file which is located in the `BabylonDependencies.framework`.
+
+### Properties with default values
+TBD
+
+### Properties without default values..
+..which depends on asyncronous events to have its value
+
+TBD
+
+### Runtime dynamic changes as `DesignLibrary`
```

Comment:
What is the value of documenting these here?
If the variable is something Reactive like `Signal`/`Property`/etc you get a strong hint 
that it will change over time.

Reply:
> strong hint 

is not documentation.

If you feel that it's overlapping too much, I can remove it from here. 
But I feel that it's not that clear how to use stuff (Current, DesignLibrary, NVL, components). Imagine for a new joiner. 

I thought you'd feel happy with it. 

Again, don't you see value on this kind of documentation? @gtsiap

Reply:
I think it will make sense at least to mention that these properties should be observed rather than be accessed via `.value`. But for that may the next section will be already a good fit.